### DISCUSSION TITLE
Chia switched from mainnet to testnet7

### DISCUSSION THREAD

**Discussion by gregzawadzki** [Chia-Network/chia-blockchain#5225] [Support]
Everything was fine until i started to play with CLI

Not sure why. I probably used `chia configure -t true`, which switched me to testnet7.
going `-t false` does not return me to `mainnet`

I asked on Reddit, but this seems to be better place for support (for reference): https://www.reddit.com/r/chia/comments/ndplfp/chia_switched_to_testnet7/

**Comment by gregzawadzki** [comment]
I compared 2 files line-by-line and returned settings manually.
Somehow my ports got switched from 8444 to 58444 and mainnet to testnet7 everywhere.
Seems to work fine. Hope will save someone time.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
